docs/tutorials/python.md

```diff
@@ -1,17 +1,19 @@
-# Tutorial: Doing python development with pixi
+# Tutorial: Doing Python development with Pixi
 
-In this tutorial we will show you how to create a simple python project with pixi. We will show some of the features that pixi provides, that are currently
+In this tutorial, we will show you how to create a simple Python project with pixi. We will show some of the features that pixi provides, that are currently
 not a part of `pdm`, `poetry` etc.
 
 ## Why is this useful?
-Pixi builts upon on the conda ecosystem, which allows you to create a python environment with all the dependencies you need. This is especially useful when you are working with multiple python interpreters and bindings to C and C++ libraries. Let's give it a go! E.g GDAL from PyPI does not have the binary C dependencies, but the conda package does.
 
-## pixi.toml and pyproject.toml
-We support two manifests formats, `pyproject.toml` and `pixi.toml`. In this tutorial we will use the `pyproject.toml` format. Because it is the most common format for python projects.
+Pixi builds upon the conda ecosystem, which allows you to create a Python environment with all the dependencies you need. This is especially useful when you are working with multiple Python interpreters and bindings to C and C++ libraries. Let's give it a go! For example, GDAL from PyPI does not have binary C dependencies, but the condom package does. However, some packages are only available through PyPI, which is why we can explicitly add those as a fallback to be solved together by pixi.
```

Comment:
Quite a few honestly. 

My employer for example almost exclusively publishes them on PyPI so far. The framework my website uses (Nikola) only has a "community package on conda" last updated over a year ago and the recommended install is pip. In my world most people try to use pip first before conda and because of some clunkiness around conda itself (despite mamba) steered away from using conda. 

But I'm definitely not an expert on this. Just my limited experience.

Reply:
Just noticed from this package that it say "condom" package, I think it should be "conda" 😃

Reply:
Oh lord. That's embarrassing.

Reply:
@JesperDramsch, yeah I got the sentence as it is the hard truth we have to concur with pixi. I was indeed joking on the name :sweat_smile:

Reply:
@ruben-arts heh, I'm too autistic to catch that sorry 😅